securedrop_client/logic.py

```diff
@@ -525,6 +530,7 @@ def logout(self):
         self.gui.logout()
 
         self.is_authenticated = False
+        self.update_sync()
```

Comment:
this can be removed-- we only want to get and show the latest last sync time:

* immediately on logout or logging in in offline mode or when the queue is paused
* every thirty seconds while offline

Reply:
But this call is needed to immediately show the latest sync time on logout, right..? I'm probably missing something here, but it could be up to 30 seconds before the `update_sync` method is called for the information to reach the UI (i.e. the QTimer which ensures this is checked every 30 seconds). This call means it's immediately displayed to the user on logout.

Reply:
my mistake, this comment was meant for your call to `update_sync` in `update_sources`

this `update_sync` actually belongs in `logout` and `login_offline_mode`